tests/e2e_istio_preinstalled.sh

```diff
@@ -33,7 +33,7 @@ declare -a tests
 
 HUB=gcr.io/istio-testing
 
-cd ${WORKSPACE}/istio
+cd ${WORKSPACE}/github.com/istio
```

Comment:
shouldn't this be istio.io/istio ?

Reply:
if `istio.io/istio` exists, yeah sure, I'll update that accordingly.

Reply:
I am not sure how you checkout istio, but usually it does go to github.com/istio

@ldemailly We are using istio.io/istio because we have a symbolic link.

Reply:
yeah currently we do checkout through github.com/istio